lib/animina_web/components/chat/chat_components.ex

```diff
@@ -125,7 +134,17 @@ defmodule AniminaWeb.ChatComponents do
     ~H"""
     <%= if @sender.id != @message.sender_id do %>
       <div class="flex justify-start gap-4  item-start text-black">
-        <.receiver_user_image user={@receiver} current_user={@sender} />
+        <.receiver_user_image
+          state={
+            if @receiver.profile_photo && is_atom(@receiver.profile_photo.state) do
```

Comment:
This cries out loud to become some sort of helper function. DRY!

As fare as I understand it we are dealing with a potential Ash bug here. There will come a time when that bug is fixed. It is so much easier than to just fix one helper function than to walk through all the other code.

Reply:
Very True , I will have a seperate issue for this